src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -69,7 +80,10 @@ const ChartsHolder = (props: {
   return (
     <>
       {!projection ? (
-        <NoCountyDetail countyId={props.countyId} stateId={props.stateId} />
+        <NoCountyDetail
+          countyId={props.county.county_url_name}
```

Comment:
This code probably only gets hit for counties, but to be safe, `props.county?.county_url_name` here too.

Reply:
Done!